ccswbs/doorknob
# Remove brightedge cookie bar and related links

# Summary of changes
Remove brightedge cookie bar and related links

## Frontend
List all significant changes to Gatsby code

[x] My changes are accessible (at minimum WCAG 2.0 Level AA)
[x] My changes are responsive and appear as expected on mobile and desktop views.

## Backend
None

# Test Plan

1. View the homepage
2. Confirm Brightedge code has been removed
